packages/office-ui-fabric-react/src/components/Panel/Panel.base.tsx

```diff
@@ -288,35 +290,43 @@ export class PanelBase extends BaseComponent<IPanelProps, IPanelState> implement
   };
 
   private _onRenderNavigationContent = (props: IPanelProps): JSX.Element | null => {
-    const { closeButtonAriaLabel, hasCloseButton } = props;
+    const { closeButtonAriaLabel, hasCloseButton, onRenderHeader = this._onRenderHeader } = props;
     const theme = getTheme();
     if (hasCloseButton) {
       // TODO -Issue #5689: Comment in once Button is converted to mergeStyles
       // const iconButtonStyles = this._classNames.subComponentStyles
       // ? (this._classNames.subComponentStyles.iconButton as IStyleFunctionOrObject<IButtonStyleProps, IButtonStyles>)
       // : undefined;
+
       return (
-        <IconButton
-          // TODO -Issue #5689: Comment in once Button is converted to mergeStyles
-          // className={iconButtonStyles}
-          styles={{
-            root: {
-              height: 'auto',
-              width: '44px',
-              color: theme.palette.neutralSecondary,
-              fontSize: IconFontSizes.large
-            },
-            rootHovered: {
-              color: theme.palette.neutralPrimary
-            }
-          }}
-          className={this._classNames.closeButton}
-          onClick={this._onPanelClick}
-          ariaLabel={closeButtonAriaLabel}
-          title={closeButtonAriaLabel}
-          data-is-visible={true}
-          iconProps={{ iconName: 'Cancel' }}
-        />
+        <React.Fragment>
+          {!this._hasCustomNavigation && onRenderHeader(this.props, this._onRenderHeader, this._headerTextId)}
```

Comment:
So when there's custom navigation, are we sure the textbook shouldn't be aligned - I think design wise, the 2nd looks more together:
Screen Shot 2019-12-05 at 4 36 09 PM

Instead of:
Screen Shot 2019-11-25 at 3 41 19 PM

Reply:
replied in Teams